oxen-rust/README.md

```
@@ -248,32 +242,40 @@ You an also increase the number of open files your system allows ulimit before r
+```
+cargo nextest run
+```
+
+Run with vanilla cargo:
 ```
 cargo test -- --test-threads=$(nproc)
 ```
 
 It can be faster (in terms of compilation and runtime) to run a specific test. To run a specific library test:
 
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)